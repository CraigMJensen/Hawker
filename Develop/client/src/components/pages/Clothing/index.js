import React from 'react';
import { CCard, CCardBody, CButton } from '@coreui/react';

function Clothing() {
  return (
    <div>
      <div className="title">Clothing</div>
      <div className="itemContainer">
        <CCard>
          <CCardBody>
            <CButton href="#">Check it out!</CButton>
          </CCardBody>
        </CCard>
      </div>
    </div>
  );
}

export default Clothing;
