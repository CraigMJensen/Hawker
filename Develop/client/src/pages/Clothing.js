import React from 'react';

const Clothing = () => {

}

export default Clothing;