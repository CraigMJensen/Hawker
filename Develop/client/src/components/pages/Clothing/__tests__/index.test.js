// Import dependencies
import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Clothing from '..';

// After each test, make sure there isnt any leftover memory that could give you false results
afterEach(cleanup);

// Declare the components you are testing
describe('Clothing component', () => {
  // verify the component is rendering properly
  it('Clothing component renders', () => {
    render(<Clothing />);
  });

  // compare snapshot versions of the DOM node structure
  it('matches snapshot DOM node structure', () => {
    const { asFragment } = render(<Clothing />);
    expect(asFragment()).toMatchSnapshot();
  });
});