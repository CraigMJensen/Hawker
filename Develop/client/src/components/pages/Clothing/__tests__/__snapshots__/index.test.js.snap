// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clothing component matches snapshot DOM node structure 1`] = `
<DocumentFragment>
  <div>
    <h2>
      Clothing
    </h2>
  </div>
</DocumentFragment>
`;
